script/script.js

```diff
@@ -0,0 +1,71 @@
+const titleEl = document.getElementById('titleEl');
+const authorEl = document.getElementById('authorEl');
+const form = document.getElementById('my-form');
+const bookList = document.querySelector('.book-list');
+
+/* new code with class */
+// let dataLink = Math.random();
+let books = [];
```

Comment:
- [ ] Great job so far 👍 Kindly note that you can use JavaScript objects and arrays but it has to be inside the JavaScript classes. It would be great if you could implement this feature as line 8 is not inside a class.